core/src/main/java/org/apache/accumulo/core/clientImpl/ScannerIterator.java

```diff
@@ -84,7 +85,8 @@ public class ScannerIterator implements Iterator<Entry<Key,Value>> {
         new ScanState(context, tableId, authorizations, new Range(range), options.fetchedColumns,
             size, options.serverSideIteratorList, options.serverSideIteratorOptions, isolated,
             readaheadThreshold, options.getSamplerConfiguration(), options.batchTimeOut,
-            options.classLoaderContext, options.executionHints);
+            options.classLoaderContext, options.executionHints,
+            (options.getConsistencyLevel().equals(ConsistencyLevel.EVENTUAL)) ? true : false);
```

Comment:
This ternary operator is redundant... the equals method already returns true or false. Also, this looks like an enum comparison and should use `==` for type safety instead of `.equals`.

Reply:
Addressed in 9ec7f5f